test/utils.mjs

```diff
@@ -339,9 +339,31 @@ export const expectTransaction = res => {
 export const depositNTransactions = async (nf3, N, ercAddress, tokenType, value, tokenId, fee) => {
   const depositTransactions = [];
   for (let i = 0; i < N; i++) {
-    depositTransactions.push(nf3.deposit(ercAddress, tokenType, value, tokenId, fee));
+    let res;
+    let count = 3; // sometimes in testnet we have issues about nonce and replacement transactions so we try again
+
+    while (count > 0) {
+      try {
+        res = await nf3.deposit(ercAddress, tokenType, value, tokenId, fee);
+        count = 0;
+      } catch (e) {
+        if (
+          e.message.includes('nonce too low') ||
+          e.message.includes('replacement transaction underpriced')
+        ) {
+          count -= 1;
+          logger.debug(`Transaction failed. Trying again...${count} tries left`);
+          await new Promise(resolving => setTimeout(resolving, 10000));
```

Comment:
I think we could create a utility function into some of our `Utils` js files to have a timeout function where we just pass over the desired `timeout` (e.g. `waitForTimeout(timeoutInMs)`), and replace it everywhere this construction is being used.